distributed/dashboard/worker_html.py

```diff
@@ -1,23 +1,70 @@
+import logging
 from .utils import RequestHandler, redirect
 
 
 class _PrometheusCollector(object):
-    def __init__(self, server, prometheus_client):
-        self.server = server
+    def __init__(self, server):
+        self.worker = server
+        self.logger = logging.getLogger("distributed.dask_worker")
+        self.crick_available = True
+        try:
+            import crick  # noqa: F401
+        except ImportError:
+            self.crick_available = False
+            self.logger.info("Digest-based metrics require crick to be installed")
```

Comment:
Could this message contain a little more context? Given that you are logging under `distributed.dask_worker` it may not be clear that this is prometheus or monitoring related. 

Perhaps including the word prometheus in here somewhere would be useful.